v8go.cc

```diff
@@ -808,6 +808,23 @@ ValuePtr NewValueIntegerFromUnsigned(IsolatePtr iso, uint32_t v) {
   return tracked_value(ctx, val);
 }
 
+RtnValue NewValueStringFromByteArray(IsolatePtr iso, const uint8_t* v, int len){
```

Comment:
This should be consistent with the Go method it is called from

```suggestion
RtnValue NewStringFromBytes(IsolatePtr iso, const uint8_t* v, int len){
```